src/ConsoleStyle.fs

```diff
@@ -0,0 +1,282 @@
+namespace MF.ConsoleStyle
+
+module private Table =
+    let private isNotEmpty seq =
+        seq
+        |> Seq.isEmpty
+        |> not
+
+    let private createRow items =
+        items
+        |> String.concat " "
+
+    let private renderSeparatorRow wordLengths =
+        let createSeparatorForWord wordLength =
+            String.replicate wordLength "-"
+
+        wordLengths
+        |> List.map createSeparatorForWord
+        |> createRow
+        |> printfn "%s"
+
+    let private getMaxWordLengthsPerColumn hasHeaders hasRows header rows =
+        let allRows =
+            match hasRows && not hasHeaders with
+            | true -> rows
+            | false -> rows |> Seq.append [header]
+
+        let getColumnWordLength row =
+            row
+            |> Seq.mapi (fun column word -> (column, word |> String.length))
+
+        allRows
+        |> Seq.collect getColumnWordLength
+        |> Seq.fold (fun maxWordLengthsPerColumn (column, wordLength) ->
+            let isColumnMaxLengthStored = maxWordLengthsPerColumn |> Map.containsKey column
```

Comment:
alternative approach: 
            maxWordLengthsPerColumn 
            |> Map.add 
                column
                (match maxWordLengthsPerColumn.TryGetValue column with
                | true, value -> max value wordLength
                | false, _ -> wordLength)

Reply:
nice :+1: 

even though I've used `tryFind` instead of `TryGetValue`